ruby/example_code/lambda/README.md

```diff
@@ -1,3 +1,92 @@
+# Lambda code examples for the AWS SDK for Ruby
+
+## Overview
+
+These code examples demonstrate how to use the AWS SDK for Ruby (v3) to manage and invoke
+AWS Lambda functions.
+
+## ⚠️ Important
+* Running this code might result in charges to your AWS account.
+* Running the tests might result in charges to your AWS account.
+*  We recommend that you grant your code least privilege. At most, grant only the minimum permissions required to perform the task. For more information, see [Grant least privilege](https://docs.aws.amazon.com/IAM/latest/UserGuide/best-practices.html#grant-least-privilege).
+* This code is not tested in every AWS Region. For more information, see [AWS Regional Services](https://aws.amazon.com/about-aws/global-infrastructure/regional-product-services).
+
+## Code examples
+
+### Single action
+
+* [Create a function](lambda_basics.rb)
+  (`CreateFunction`)
+* [Delete a function](lambda_basics.rb)
+  (`DeleteFunction`)
+* [Get a function](lambda_basics.rb)
+  (`GetFunction`)
+* [Invoke a function](lambda_basics.rb)
+  (`Invoke`)
+* [List functions](lambda_basics.rb)
+  (`ListFunctions`)
+* [Update function code](lambda_basics.rb)
+  (`UpdateFunctionCode`)
+* [Update function configuration](lambda_basics.rb)
+  (`UpdateFunctionConfiguration`)
+
+### Scenario
+
```

Comment:
Code examples that show you how to accomplish a specific task by calling multiple functions within the same service.